src/services/jwt.ts

```diff
@@ -1,10 +1,11 @@
+import { JWT_SECRET } from "@config/app"
 import * as jwt from "jsonwebtoken"
 
 // TODO: replace network by uri of the server
 // the uri will embed the network, ie: graphql.mainnet.server.io
 // and provide more information than just the network
-export const createToken = ({ uid, network }) =>
-  jwt.sign({ uid, network }, process.env.JWT_SECRET, {
+export const createToken = ({ uid, network }) => {
```

Comment:
is this a new rule? should we use return in one line functions?

Reply:
one liners are okay, but this a multi-line function even though it returns a single thing. I think having return here is clearer.